franklin/core/views.py

```diff
@@ -1,12 +1,88 @@
 import logging
 
 from rest_framework.decorators import api_view
+from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
+from rest_framework.views import APIView
+from rest_framework_social_oauth2 import views as social_oauth_views
+from social.pipeline.partial import partial
 
 logger = logging.getLogger(__name__)
 
 
+class ConvertTokenView(social_oauth_views.ConvertTokenView, APIView):
+    """ Login user """
+    permission_classes = (AllowAny,)
+
+    def post(self, request, *args, **kwargs):
+        """ 
+        Exchange social token for franklin oAuth access token
+        ---
```

Comment:
django-rest-swagger wasn't auto detecting the endpoints created by django-rest-framework-social-oauth2. For now, I'm going to extend each of their views so we have properly documented endpoints. For now it'll just be this endpoint and this one is important for dashboard integration. 

I'm going to look into fixing this in django-rest-framework-social-oauth2 as that library is supposed to play nicely with django-rest-framework.